### DISCUSSION TITLE
Volumetric lighting

### DISCUSSION THREAD

**Discussion by srink007** [google/model-viewer#2996] [Q&A] [answered]
hello everyone,
any possibilty to have volumetric lighting for GLFT/GLB files.
I am a starter, sorry for noob question

**Comment by WadeMorrison** [comment, marked as answer]
Hello!
Are you wanting something along the lines of "god rays"? I am unsure if model-viewer supports something like that as it can be pretty process-intensive. I think you'd be better off faking it with some god ray cards. I know there is a volume extension that GLTF supports now, but that would be more for objects that are like glass, acrylic, water, etc. Not an environment setting.

**Reply by srink007** [reply]
Thank you wade.
I can understand now.
Also I have some other doubts
Related to baking.
Can you help ?
Like baking textures

**Reply by WadeMorrison** [reply]
Depending on what software you use, I for sure can help! I use blender, and am pretty rusty with Maya. There are a ton of tutorials on youtube as well about baking textures.

**Reply by srink007** [reply]
Thank you very much brother , i am using Cinema 4d , and for gltb exports i import to Unreal engine , but i am not satisfied with results, i mean there must be a way to do it right. is there any other way to reach you . than github , twitter or some other stuff

**Reply by WadeMorrison** [reply]
You could add me on discord if you'd like? WadeMorrison#7258

**Reply by srink007** [reply]
Hi brother, sent you a req in disc
